index.js

```diff
@@ -58,4 +60,4 @@ function initialiseRoutes() {
 server.listen(8080, function () {
     initialiseRoutes();
     LogEventDispatcher.log(server.name + ' listening at ' + server.url);
-});
```

Comment:
I have no idea why it says this is removed, if you click view in the top right corner, it's still there...